### PR TITLE
feat: Add width/height overrides

### PR DESCRIPTION
**Pull request by scttcper**
Allows the request to override the style's default height/width

### CODE REVIEW

File: tests/renderServer.spec.ts [hunk 1]
```diff
@@ -44,6 +44,26 @@ describe('renderServer', () => {
...
+          data: [0, 5, 30],
+        },
+      };
+
+      const app = renderServer(config);
+      const resp = await supertest(app).post('/render').send(renderData);
+
+      expect(resp.status).toBe(200);
+      expect(resp.body).toMatchImageSnapshot();
+      expect(resp.headers['x-config-version']).toBe('1.0-test');
```
Comment (evanpurkhiser) [inline tests/renderServer.spec.ts:64]: 👀

Comment (evanpurkhiser) [reply tests/renderServer.spec.ts:64]: did i add this config version that's neat

File: src/types.ts [hunk 2]
```diff
@@ -26,7 +26,10 @@ export type RenderDescriptor<D extends string = string> = {
 /**
  * Maps style keys to style descriptor configuration
  */
-export type RenderConfig<D extends string = string> = Map<D, RenderDescriptor<D>>;
+export type RenderConfig<D extends string = string> = Map<
+  D,
+  Readonly<RenderDescriptor<D>>
```
Comment (evanpurkhiser) [inline src/types.ts:31]: 👍

**Review by evanpurkhiser — APPROVED** [review]

File: tests/renderServer.spec.ts (hunk 1 above)
Comment (scttcper) [reply tests/renderServer.spec.ts:64]: oh i probably don't care about that in this test

**Pull request merged**